prepare-commit-msg

```diff
@@ -5,11 +5,16 @@ set -eu
 append_coauthors() {
     COMMIT_MSG_FILE=$1
 
-    tmpfile="$COMMIT_MSG_FILE"-temp
-    sed -e '/^Co-authored-by/d' $COMMIT_MSG_FILE >"$tmpfile"
-    mv "$tmpfile" "$COMMIT_MSG_FILE"
+    if grep -q -i -e "Co-authored-by" "$COMMIT_MSG_FILE"; then
+      echo 'Co-author already supplied'
+    else
+      tmpfile="$COMMIT_MSG_FILE"-temp
+      sed -e '/^Co-authored-by/d' $COMMIT_MSG_FILE >"$tmpfile"
```

Comment:
What's the point of this line if "Co-authored-by" doesn't appear anywhere in the message?

Reply:
Which line are you referring to?  This is the existing code, I just moved it under the else clause.

Reply:
By moving this existing code under the else clause, you have changed what it does. Previously, it deleted any existing `Co-authored-by` lines. If this else clause is only run when we've already determined, using `grep`, that there aren't any `Co-authored-by` lines then what is the point of this line?